tests/conftest.py

```diff
@@ -318,6 +313,11 @@ def additional_test_data():
 
 @pytest.fixture(scope="function")
 def example_data_model_file() -> BinaryIO:
-    """An example spreadsheet in towns-fund schema format."""
+    """
+    An example spreadsheet in towns-fund schema format.
+
+    This format of Excel file is not really used for anything other than testing - this fixture should not be used
+    to replicate real form ingest behaviour
+    """
```

Comment:
We now have permission to store the actual R3 templates in the repo with dummy data in. Currently we have a bunch of R4 templates + dummy data, and I'm intending to put some dummy data in a legitimate R3 template and write a test with it. At that point, I suspect we can remove this entirely as it will be redundant.